YMChat/YMTheme.swift

```diff
@@ -36,5 +38,6 @@ public class YMTheme: NSObject, Encodable {
         try container.encodeIfPresent(self.botIcon, forKey: .botIcon)
         try container.encodeIfPresent(self.botDescription, forKey: .botDesc)
         try container.encodeIfPresent(self.botClickIcon, forKey: .botClickIcon)
+        try container.encodeIfPresent(self.chatBotTheme, forKey: .chatBotTheme)
```

Comment:
@rohit-131 add lowerCase to this.

`self.chatBotTheme.lowercased()`

Reply:
As discussed, @rohit-131 will make this change at widget level.

Reply:
Will update the change in widget side to handle this